caf-audit-service-container/README.md

```diff
@@ -7,19 +7,25 @@ This is a docker container for the [CAF Audit Web Service](https://github.com/CA
 #### Environment Variables
 
 ##### CAF\_ELASTIC\_HOST\_AND\_PORT
-Comma separated list of Elasticsearch HOST:PORT value pairs. e.g. 192.168.56.10:9200,192.168.56.20:9200
+Comma separated list of Elasticsearch HOST:PORT value pairs. e.g. 192.168.56.10:9200,192.168.56.20:9200. Default value is elastic search host value constructed from alternate configurations CAF_ELASTIC_HOST_VALUES and CAF_ELASTIC_PORT_VALUE.
+
+##### CAF\_ELASTIC\_HOST\_VALUES
+This is the alternative configuration with comma seperated list of Elasticsearch host names. eg. localhost Default value: localhost.
```

Comment:
@dermot-hardy , Could you confirm this default value for hostname?

Reply:
@thilagasanthosh Start the container and confirm these values before sending for review.

Reply:
No default value for the CAF_ELASTIC_HOST_VALUES and CAF_ELASTIC_PORT_VALUE. If there is no elastic search host configured, AuditConfigurationException exception will be thrown from code.